docs: update documentation for recently implemented features (#902)
sct/overseerr: docs/support/faq.md

### I receive 409 or 400 errors when requesting a movie or tv show!

> **A:** Verify your are running Radarr and Sonarr v3. Overseerr was developed for v3 and is not currently backward compatible.

Reviewer: ```suggestion
**A:** Verify you are running Radarr and Sonarr v3. Overseerr was developed for v3 and is not currently backward-compatible.
```

Author: Nice catch

### Can I make 4K requests?

> **A:** Yes! When adding your 4K Sonarr/Radarr server, tick the `4K Server` checkbox. You also need to tick the `Default Server` checkbox if the Sonarr/Radarr server you're adding is the default one you want your requests to go to for 4K content (there needs to be a default Sonarr/Radarr server for **both** 4K **and** non-4K content).

Reviewer: ```suggestion
**A:** Yes! When adding your 4K Sonarr/Radarr server, tick the `4K Server` checkbox. You also need to tick the `Default Server` checkbox if it is the default server you would like to use for 4K content requests. (To enable 4K requests, there need to be default Sonarr/Radarr servers for both 4K content **and** non-4K content.)
```

### Can I create local users in Overseerr?

> **A:** Head to **Settings -&gt; Users** and hit **Create Local User**. Keep in mind that local user accounts need a valid email address.

Reviewer: ```suggestion
**A:** Head to **Users** and hit **Create Local User**. Keep in mind that local user accounts need a valid email address.
```

### Is is possible to set user roles in Overseerr?

> **A:** The user roles can be set for each user individually on the **Users** page. The list of permissions is one that is still growing, so if you have any suggestions, [make a feature request](https://github.com/sct/overseerr/issues/new/choose) on GitHub.

Reviewer: ```suggestion
**A:** User roles can be set for each user on the **Users** page. The list of assignable permissions is one that is still growing, so if you have any suggestions, [make a feature request](https://github.com/sct/overseerr/issues/new/choose) on GitHub.
```

## Requests

### I approved a requested movie and Radarr didn't search for it!

> **A:** Check your minimum availability in Radarr. If an added item does not meet the minimum availability, no search will be performed. Also verify that Radarr did not search for it by checking the Radarr logs. Lastly, verify the item was not already being monitored by Radarr.

Reviewer: ```suggestion
**A:** Check the minimum availability setting in your Radarr server. If a movie does not meet the minimum availability requirement, no search will be performed. Also verify that Radarr did not perform a search, by checking the Radarr logs. Lastly, verify that the item was not already being monitored by Radarr prior to approving the request.
```
